YololCompetition/Services/Challenge/DbChallenges.cs

```diff
@@ -24,11 +25,13 @@ private enum Status
 
         private readonly IDatabase _database;
         private readonly Configuration _config;
+        private readonly IMessages _messages;
 
-        public DbChallenges(IDatabase database, Configuration config)
+        public DbChallenges(IDatabase database, Configuration config, IMessages messages)
```

Comment:
This takes `IMessages` as a parameter, but doesn't seem to use it any more.

Reply:
true true